models/core/staging/core__stg_claims_encounter.sql

```diff
@@ -0,0 +1,36 @@
+{{ config(
+     enabled = var('claims_enabled',var('tuva_marts_enabled',False))
+   )
+}}
+
+-- *************************************************
+-- This dbt model creates the encounter table in core.
+-- *************************************************
+
+select
```

Comment:
This is a comment for all of the stage tables for both clinical and claims. We need to add data type casting in every stage to ensure the union works properly when there is data from both source types. Especially since the core table is unioning with select *.